marklogic-data-hub/gradle.properties

```diff
@@ -52,6 +52,12 @@ mlUseRoxyTokenPrefix=false
 ## These values are used by the testing infrastructure.
 mlHubUserRole=data-hub-role
 mlHubUserName=data-hub-user
+mlHubAdminRole=hub-admin-rule
+mlHubAdminUser=hub-admin-user
```

Comment:
is this a typo. Should it be hub-admin-role?

Reply:
yes most definitely a typo